### PR TITLE
Add get started page 

### PR DESCRIPTION
**Pull request by georgeschena**
Add a get started page to the GovWifi product page site to allow users to not feel overwhelmed when signing up. By providing this get started page it gives the user information they may need in order to sign up to GovWifi.

### CODE REVIEW

File: source/getstarted.html.erb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    </li>
+    <li class="breadcrumbs__item breadcrumbs__item--active" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+      <a href="#main" itemprop="item"><span itemprop="name">Get started</span></a>
+    </li>
+  </ol>
+</nav>
+<div class="container">
+  <div class="grid-row">
+    <div class="column-two-thirds column-minimum">
+      <h1 class="heading-large" style="padding-bottom: 15px" >Get started</h1>
```
Comment (emileswarts) [inline source/getstarted.html.erb]: Is this inline style to make it line up?
Nothing we can use with predefined styles?

File: source/getstarted.html.erb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        <div class="task-list-items" style="padding-bottom: 15px">
+          <p>Create an admin account so you can:</p>
+          <ul class="list list-bullet">
+            <li> add new IP addresses </li>
+            <li> add new team members </li>
+            <li> find the connection details for GovWifi radius servers </li>
+          </ul>
+        </div>
+        <h2 class="task-list-section">
+          <span class="task-list-section-number">3.</span> Sign a memorandum of understanding
```
Comment (emileswarts) [inline source/getstarted.html.erb]: These spaces within the html tags will create a unwanted whitespace?
```
</span> Sign...
```

File: source/getstarted.html.erb (hunk 1 above)
Comment (georgeschena) [reply source/getstarted.html.erb]: The predefined styles weren't applying. So I used inline.

Comment (georgeschena) [reply source/getstarted.html.erb]: I was to give spacing between the different sections

Comment (emileswarts) [reply source/getstarted.html.erb]: Does it look awful if we didn't apply the inline styles?
A bit worried that we are missing predefined styles that can do this for us.

Comment (georgeschena) [reply source/getstarted.html.erb]: Ive used the predefined styles now. It does not look bad imo, was just being a perfectionist.

Comment (georgeschena) [reply source/getstarted.html.erb]: This is using the predefined styles @emileswarts 

screen shot 2018-10-09 at 12 20 55

**Pull request merged**